sql/api/src/main/scala/org/apache/spark/sql/catalyst/analysis/noSuchItemsExceptions.scala

```diff
@@ -89,18 +82,10 @@ class NoSuchNamespaceException private(
     this(errorClass = "SCHEMA_NOT_FOUND",
       Map("schemaName" -> quoteNameParts(namespace)))
   }
-
-  def this(message: String, cause: Option[Throwable] = None) = {
```

Comment:
@MaxGekk @dongjoon-hyun @aokolnychyi this brings a similar breaking change as SPARK-43041 reported